numba/tests/test_unicode.py

```diff
@@ -1295,6 +1295,18 @@ def test_slice3_error(self):
                                          cfunc(s, sl),
                                          "'%s'[%d:%d:%d]?" % (s, i, j, k))
 
+    def test_slice_ascii_flag(self):
+        """
+        Make sure ascii flag is False when ascii and non-ascii characters are
+        mixed in output of Unicode slicing.
+        """
+        @njit
+        def f(s):
+            return s[::2]._is_ascii
```

Comment:
```suggestion
            return s[::2]._is_ascii, s[1::2]._is_ascii
```
Suggest adding and testing the converse too.